sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job.py

```diff
@@ -319,6 +323,8 @@ def _from_rest_object(  # pylint: disable=too-many-return-statements
             if obj.properties.job_type == RestJobType.AUTO_ML:
                 return AutoMLJob._load_from_rest(obj)
             if obj.properties.job_type == RestJobType_20240101Preview.FINE_TUNING:
+                if obj.properties.properties.get("azureml.enable_distillation", False):
```

Comment:
this seems contradictory. Should it be 

```suggestion
                if obj.properties.properties.get("azureml.enable_distillation", True):
```

Reply:
Since we are using the FT overload, for FT jobs, "azureml.enable_distillation" would not be there so if the default is True, every FT job would be labeled as distillation